sagemaker-featurestore/feature_store_object_detection_ground_truth.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        for entry in open(filename, 'r'):
</pre><p> </p><p>Is it possible to rename and clarify what i, k, j means here? I've never used GroundTruth before, so these symbols look confusing to me. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2822/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>